RELEASE_NOTES.md

```diff
@@ -6,6 +6,7 @@
 - added: `git machete help <command>` now accepts command aliases
 - fixed: removed stack trace when terminating `git machete` command prompt with `Ctrl+D`
 - added: support for Python 3.11
+- added: machete config key `machete.traverse.push` that controls `--push` flag of `traverse` command
```

Comment:
But... but... as discussed, I don't think the config controls the **flag**, it controls the **behavior** (which also happens to be controlled by the flag). And if anything, it's the flag that has the higher priority than the config, so one could even argue that it's `--push` that controls `machete.traverse.push` and not the other way round 😅

Reply:
So... the current formulation is deeply confusing ☹️

Reply:
Oh, a commit must have disappeared, i remember that i changed it